akka-actor-tests/src/test/scala/akka/AkkaVersionSpec.scala

```diff
@@ -59,7 +59,7 @@ class AkkaVersionSpec extends WordSpec with Matchers {
     }
 
     "succeed if Akka version is SNAPSHOT" in {
-      AkkaVersion.require("AkkaVersionSpec", "2.5.6", "2.5-SNAPSHOT")
+      AkkaVersion.require("AkkaVersionSpec", "2.5.6", "2.5.16")
```

Comment:
sure? the test is specifically about testing -SNAPSHOT

Reply:
Sorry, I had changed this to have a specific version in my OSGi Container to test with. I have now reverted that change.

Reply:
No worries, thanks!

Reply:
this is still remaining

Reply:
I have now changed that. However, I just double checked my git logs and it seems that I have never touched that file. I did at one point set 2.5.-SNAPSHOT to 2.5.16-OSGI. Nevermind...mysteries og Git :)